src/pytest_benchmark/plugin.py

```diff
@@ -391,8 +396,25 @@ def pytest_runtest_setup(item):
                 raise ValueError("benchmark mark can't have %r keyword argument." % name)
 
 
+def get_report_backend(config):
+    storage = config.getoption("benchmark_storage")
+    if storage.startswith("file://"):
+        config.option.__dict__["benchmark_storage"] = storage[len("file://"):]
+        return FileReportBackend(config)
+    elif storage.startswith("elasticsearch+"):
+        hosts, index, doctype, project_name = parse_elasticsearch_storage(storage[len("elasticsearch+"):])
+        config.option.__dict__["benchmark_elasticsearch_hosts"] = hosts
+        config.option.__dict__["benchmark_elasticsearch_index"] = index
+        config.option.__dict__["benchmark_elasticsearch_doctype"] = doctype
+        config.option.__dict__["benchmark_project_name"] = project_name
+        config.option.__dict__["benchmark_autosave"] = get_tag(project_name)
+        return ElasticReportBackend(config)
+    else:
```

Comment:
Regarding https://github.com/ionelmc/pytest-benchmark/pull/58#issuecomment-245860797, can we also support legacy-style paths (auto-convert them to `file://<value>` if value doesn't contain `://`)? Tests are failing anyway because of this :)